libs/blocks/global-navigation/utilities/onImsReady.js

```diff
@@ -0,0 +1,31 @@
+let ready = false;
+let existingCall = null;
+const onInstance = 'onImsLibInstance';
+
+const onImsReady = (timeout = 3000) => {
```

Comment:
where is this imported / called?

Reply:
It's currently not used anywhere, but we want to avoid using `adobeid.onReady` callback given the async nature of blocks. The utility can be used for personalisation, API calls that need user information, tokens etc.